include/sender_cadmium.hpp

```diff
@@ -106,25 +108,27 @@ class Sender {
 
         // external transition
         void external_transition(
-		                TIME e,
-	                        typename make_message_bags<input_ports>::type mbs) { 
+	    TIME e,
```

Comment:
Line breaking and intention issue: Would suggest not to have TIME e in the same column as void external_transition (in the previous line). 
"make_message_bags<input_ports>::type mbs) { " --> this can go along with typename, it will be less than 80 characters. 
i.e., typename make_message_bags<input_ports>::type mbs) {

Reply:
Resolved

Reply:
Suggestion on function and breaking its parameters (for all header files):

In our code, we split the line as below, 
![image](https://user-images.githubusercontent.com/44588557/62562563-5e910400-b84f-11e9-8395-6bc31a479260.png)

And professor was saying that it was good. 

I believe the current code of your's is correct as well (but I would suggest to double check just in case).